grpcurl.hcl

```diff
@@ -0,0 +1,17 @@
+description = "Inspect and call gRPC services"
+test = "grpcurl --version"
+binaries = ["grpcurl"]
+
+darwin {
+  source = "https://github.com/fullstorydev/grpcurl/releases/download/v${version}/grpcurl_${version}_osx_${xarch}.tar.gz"
```

Comment:
Alas there is no arm64 Darwin build. Can you `s,${arch},x86_64,` so it will work on M1 Macs?

Reply:
It doesn't look like there are arm64 releases: https://github.com/fullstorydev/grpcurl/releases/

Reply:
my bad. i read the opposite of what you were saying. ignore me.

Reply:
fixed, apologies I didn't see this.